packages/flame/pubspec.yaml

```diff
@@ -17,6 +17,8 @@ dev_dependencies:
   dart_code_metrics: ^4.1.0
   dartdoc: ^0.42.0
   mocktail: ^0.1.4
+  canvas_test_helpers: ^0.1.0
+  flame_test_helpers: ^0.1.0
```

Comment:
👀  Our tests wont pass as long as this isnt released right? Maybe path depenedency and just fix it release time?

Reply:
Fixed, it won't work automagically in the publish script though